Day5/index.js

```diff
@@ -0,0 +1,202 @@
+      displayHSV(myArray2);
+      $('#color-field').css( 'background-color' , `rgb(${r},${g},${b})` );
+    }
+    else {
+      alert('The b value must be [0, 255] value.');
+    }
+  } );
+
+  $('#hsl-h-value').on( 'input' , () => {
+    let tempH = parseFloat( $('#hsl-h-value').val() );
+    let tempS = parseFloat( $('#hsl-s-value').val() );
+    let tempL = parseFloat( $('#hsl-l-value').val() );
+    if( (tempH >= 0) && (tempH <= 1) ) {
+      let myArray = hslToRgb(tempH, tempS, tempL);
+      r = Math.floor( myArray[0] );
+      g = Math.floor( myArray[1] );
+      b = Math.floor( myArray[2] );
+      displayRGB( myArray );
+      let myArray2 = rgbToHsv(r, g, b);
+      displayHSV(myArray2);
+      $('#color-field').css( 'background-color' , `rgb(${r},${g},${b})` );
+    }
+    else {
+      alert('The h value of hsl must be [0, 1] value.');
+    }
+  } );
+
+  $('#hsl-s-value').on( 'input' , () => {
+    let tempH = parseFloat( $('#hsl-h-value').val() );
+    let tempS = parseFloat( $('#hsl-s-value').val() );
+    let tempL = parseFloat( $('#hsl-l-value').val() );
+    if( (tempS >= 0) && (tempS <= 1) ) {
+      let myArray = hslToRgb(tempH, tempS, tempL);
+      r = Math.floor( myArray[0] );
+      g = Math.floor( myArray[1] );
+      b = Math.floor( myArray[2] );
+      displayRGB( myArray );
+      let myArray2 = rgbToHsv(r, g, b);
+      displayHSV(myArray2);
+      $('#color-field').css( 'background-color' , `rgb(${r},${g},${b})` );
+    }
+    else {
+      alert('The s value of hsl must be [0, 1] value.');
+    }
+  } );
+
+  $('#hsl-l-value').on( 'input' , () => {
+    let tempH = parseFloat( $('#hsl-h-value').val() );
+    let tempS = parseFloat( $('#hsl-s-value').val() );
+    let tempL = parseFloat( $('#hsl-l-value').val() );
+    if( (tempL >= 0) && (tempL <= 1) ) {
+      let myArray = hslToRgb(tempH, tempS, tempL);
+      r = Math.floor( myArray[0] );
+      g = Math.floor( myArray[1] );
+      b = Math.floor( myArray[2] );
+      displayRGB( myArray );
+      let myArray2 = rgbToHsv(r, g, b);
+      displayHSV(myArray2);
+      $('#color-field').css( 'background-color' , `rgb(${r},${g},${b})` );
+    }
+    else {
+      alert('The l value of hsl must be [0, 1] value.');
+    }
+  } );
+
+  $('#hsv-h-value').on( 'input' , () => {
+    let tempH = parseFloat( $('#hsv-h-value').val() );
+    let tempS = parseFloat( $('#hsv-s-value').val() );
+    let tempV = parseFloat( $('#hsv-v-value').val() );
+    if( (tempH >= 0) && (tempH <= 1) ) {
+      let myArray = hsvToRgb(tempH, tempS, tempV);
+      r = Math.floor( myArray[0] );
+      g = Math.floor( myArray[1] );
+      b = Math.floor( myArray[2] );
+      displayRGB( myArray );
+      let myArray2 = rgbToHsl(r, g, b);
+      displayHSL(myArray2);
+      $('#color-field').css( 'background-color' , `rgb(${r},${g},${b})` );
+    }
+    else {
+      alert('The h value of hsv must be [0, 1] value.');
+    }
+  } );
+
+  $('#hsv-s-value').on( 'input' , () => {
```

Comment:
This is so duplicate. As you can see, every `.on` methods are doing the same thing, just the element's id is different. Remove all duplicate codes and put it in one method called `setup()`.
```
setup('#hsv-h-value');
setup('#hsv-s-value');
setup('#hsv-v-value');

setup('#rgb-r-value');
setup('#rgb-g-value');
setup('#rgb-b-value');
```